packages/cozy-procedures/src/components/form/getInputProps.js

```diff
@@ -0,0 +1,64 @@
+/*eslint no-unused-vars: off*/
+const getInputProps = (
+  {
+    readonly,
+    autofocus,
+    formContext,
+    options = {},
+    rawErrors,
+    schema,
+    t,
+    f,
+    ...otherProps
+  },
+  widget = 'input'
+) => {
+  const inputProps = {
+    readOnly: readonly,
+    autoFocus: autofocus,
+    ...otherProps
+  }
+
+  if (widget === 'select') {
+    inputProps.options = options
+    return inputProps
+  }
+
+  // If options.inputType is set use that as the input type
+  if (options.inputType) {
+    inputProps.type = options.inputType
+    // If the schema is of type number or integer, set the input type to number
+  } else if (!inputProps.type) {
+    if (schema.type === 'number') {
+      inputProps.type = 'number'
+      // Setting step to 'any' fixes a bug in Safari where decimals are not
+      // allowed in number inputs
+      inputProps.step = 'any'
+    } else if (schema.type === 'integer') {
+      inputProps.type = 'number'
+      // Since this is integer, you always want to step up or down in multiples
+      // of 1
+      inputProps.step = '1'
```

Comment:
I don't get the difference between `integer` and `number`?

Reply:
I think that `integer` doesn't allow point numbers but I don't see any difference :thinking: